tools/dh_eoscontent

```diff
@@ -106,7 +105,7 @@ sub app_appdata_file {
 
 foreach my $package (@{$dh{DOPACKAGES}}) {
 	my $tmp=tmpdir($package);
-	my $app_id=package_eos_app_id($package);
+	my $app_id=$package;
```

Comment:
That's not correct. The package name is often something like `totem` while the app ID is something like `org.gnome.Totem`. Indeed, `apt-cache show totem`:
```
Package: totem
...
Eos-Appid: org.gnome.Totem
Eos-Core-App: yes
```